e2e/cypress/integration/enterprise/ldap_group/team_and_channel_assign_roles_spec.js

```diff
@@ -136,7 +135,7 @@ describe('System Console', () => {
         });
 
         // # Save the setting and navigate back to page
-        saveAndNavigateBackTo(teamName, 'teams');
+        saveAndNavigateBackTo(teamName, teamDisplayName, 'teams');
```

Comment:
Test failed on me here.
<img width="674" alt="Screen Shot 2020-10-23 at 6 06 20 PM" src="https://user-images.githubusercontent.com/5334504/96995233-74780b00-1560-11eb-96ff-c0f17d15a270.png">

It passed when I removed forced click and not using `.then`, like
```javascript
        // # Remove "board" group
        cy.get('.group-row').eq(0).scrollIntoView().should('be.visible').within(() => {
            cy.get('.group-name').should('have.text', groupDisplayName);
            cy.get('.group-actions > a').should('have.text', 'Remove').click();
        });

        // * Assert that the group was removed successfully
        cy.get('#groups-list--body').should('be.visible').contains('No groups specified yet');

        // # Save the setting and navigate back to page
        saveAndNavigateBackTo(teamName, teamDisplayName, 'teams');

        // * Assert that the group was removed successfully
        cy.get('#groups-list--body').scrollIntoView().should('be.visible').contains('No groups specified yet');
```

Reply:
@saturninoabril updated